app/views/admin/schedules/_event.html.haml

```diff
@@ -1,14 +1,18 @@
 - cells_length = event.event_type.length / EventType::LENGTH_STEP
 / this height fits the room cells
-- height = (cells_length * 58) - 23
+- compact_grid = EventType::LENGTH_STEP < 15
```

Comment:
I wonder if we really need to have two cases or we could just generalise. The problem of having bigger cells is that the admin schedule page gets to long and hard to manage. On the other hand if they are too small you can see only the few first characters of the event title and that makes really difficult to know what event you are scheduling. That was the reason what we keep it as big as it is know.

So maybe making it always compact and showing the event title when you put the mouse over the event could be a solution for both problems. :thinking: 

@hennevogel @ChrisBr @differentreality ideas?